### PR TITLE
Add Authors to web pages

### PR DESCRIPTION
**Pull request by sandrobonazzola**
Changes proposed in this pull request:

- Add Authors to web pages

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @sandrobonazzola 

This pull request needs review by: @oVirt/ovirt-site

### CODE REVIEW

**Comment by sandrobonazzola** [comment]
Example:
Screenshot_2021-05-21 Communication

File: data/authors.yml [hunk 1]
```diff
@@ -15,8 +15,6 @@ example:
     when adding your own authors. All fields are optional, so only
     include the ones that apply.
 
-wikisysop:
```
Comment (duck-rh) [inline data/authors.yml:18]: It was high time we got rid of this, thanks.

**Review by duck-rh — APPROVED** [review]
Minor remarks but LGTM. it looks very nice.

File: source/_layouts/default.html [hunk 2]
```diff
@@ -10,6 +10,44 @@
...
+
+        {% unless page.hide_metadata %}
+        {% if page.authors %}
+          <header class="post-header col-md-4 pad-sides well well-lg">
+            <h2 class="post-title entry-title">Authors:</h2>
+            <header class="post-meta">
+              {% for author_hash in page.authors %}
+              {% assign author = site.data.authors[author_hash] %}
+              {% unless author %}
+                {{ "You're missing author " | append: author_hash | raise_error }}
```
[the remark this answers is not in the document]
Comment (sandrobonazzola) [reply source/_layouts/default.html]: thanks :-)

File: source/_layouts/default.html [hunk 3]
```diff
@@ -10,6 +10,44 @@
...
+                  src="https://secure.gravatar.com/avatar/{{ author.gravatar }}?s=32&amp;d=mm&amp;r=g"
+                  srcset="https://secure.gravatar.com/avatar/{{ author.gravatar }}?s=64&amp;d=mm&amp;r=g 2x"
+                  class="avatar avatar-32 photo"
+                  height="32"
+                  width="32"
+                  loading="lazy">
+              </div>
+              <span class="byline">
+                <section class="entry-meta">
+                  {% if author.twitter != null %}
```
[the remark this answers is not in the document]
Comment (sandrobonazzola) [reply source/_layouts/default.html]: done

File: source/_layouts/default.html [hunk 4]
```diff
@@ -10,6 +10,44 @@
...
       <section id="page" class="page">
         {% include nav-breadcrumbs.html %}
+
+        {% unless page.hide_metadata %}
+        {% if page.authors %}
+          <header class="post-header col-md-4 pad-sides well well-lg">
+            <h2 class="post-title entry-title">Authors:</h2>
+            <header class="post-meta">
+              {% for author_hash in page.authors %}
+              {% assign author = site.data.authors[author_hash] %}
```
[the remark this answers is not in the document]
Comment (sandrobonazzola) [reply source/_layouts/default.html]: used author_id

File: source/_layouts/default.html [hunk 5]
```diff
@@ -10,6 +10,44 @@
     <main id="page-wrap" class="page-wrap" aria-label="Content">
       <section id="page" class="page">
         {% include nav-breadcrumbs.html %}
+
+        {% unless page.hide_metadata %}
```
[the remark this answers is not in the document]
Comment (sandrobonazzola) [reply source/_layouts/default.html:13]: fixed indentation, thanks for noticing

**Comment by duck-rh** [comment]
@sandrobonazzola maybe others can comment about the feature itself but on my side I see no problem merging this PR.

**Comment by sandrobonazzola** [comment]
Thanks for the review, merging.

**Pull request merged**